src/main/webapp/app/exercises/programming/manage/grading/programming-exercise-configure-grading.component.html

```diff
@@ -194,20 +205,38 @@ <h2 class="mb-1 fw-medium" jhiTranslate="artemisApp.programmingExercise.configur
                                     </select>
                                 </ng-template>
                             </ngx-datatable-column>
-                            <ngx-datatable-column name="Is Active" prop="active" [sortable]="false">
-                                <ng-template ngx-datatable-header-template>
-                                    <span class="datatable-header-cell-wrapper">
-                                        <span class="datatable-header-cell-label bold">Is Active</span>
+                            <ngx-datatable-column name="Points" prop="points" [comparator]="comparePoints">
+                                <ng-template ngx-datatable-header-template let-sortFn="sortFn">
+                                    <span class="datatable-header-cell-wrapper" (click)="sortFn()">
+                                        <span class="datatable-header-cell-label bold sortable">{{
+                                            'artemisApp.programmingExercise.configureGrading.gradingTableHeader.resultingPoints' | artemisTranslate
+                                        }}</span>
+                                        <fa-icon [icon]="iconForSortPropField('testCases', 'points')"></fa-icon>
+                                        <fa-icon
+                                            [icon]="faQuestionCircle"
+                                            class="text-secondary ms-2"
+                                            [ngbTooltip]="'artemisApp.programmingExercise.configureGrading.help.points' | artemisTranslate"
+                                            placement="bottom"
+                                            container="body"
+                                        ></fa-icon>
                                     </span>
                                 </ng-template>
-                                <ng-template ngx-datatable-cell-template let-value="value">
-                                    {{ value }}
+                                <ng-template ngx-datatable-cell-template let-row="row">
+                                    <input
+                                        class="table-editable-field__input form-control me-2"
+                                        [value]="testCasePoints[row.testName]"
+                                        type="text"
+                                        [disabled]="true"
+                                        [readOnly]="true"
+                                    />
```

Comment:
I don't see the sense behind adding an input that is always disabled. Then you can just write in a cell like the former "Is active" column or the "Type" column, right?